std_msgs/Empty empty
---
sensor_msgs/Image camera_image
